content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -105,6 +105,33 @@ in your **Hosted endpoints** and insert the following data into the pop-up form:
 When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
 display your signing secret.
 
+Configuration tab
+-----------------
+
+.. _stripe/manual_capture:
+
+Place a hold on a card
+~~~~~~~~~~~~~~~~~~~~~~
+
+Stripe allows you to **capture an amount manually** instead of having an immediate capture. This
+feature earmarks an amount on the customer's account for a determined period of time that depends on
+the payment method used.
+
+To set it up, enable the :guilabel:`Capture Amount Manually` option on Odoo, as explained in the
+:ref:`payment acquirers documentation <payment_acquirers/capture_amount>`.
+
+.. warning::
+   Some payment methods, unlike cards, don't support manual capture. When the manual capture feature 
+   is activated, your customers won't be able to proceed with their payment if unsupported payment 
+   methods are selected.
```

Comment:
You might need to check the 100th character I didn't check it on PyCharm this time
```suggestion
.. warning::
When the manual capture feature is activated, your customers won't be able to proceed with any payment if unsupported payment methods are selected.
```